Dockerfile

```diff
@@ -17,13 +23,21 @@ WORKDIR $APP_HOME
 ADD Gemfile ./
 ADD Gemfile.lock ./
 
-RUN apk --update add --virtual build-dependencies build-base ruby-dev openssl-dev \
-    postgresql-dev zlib-dev libxml2-dev libxslt-dev libffi-dev && \
-    gem install -N bundler && \
-    cd $APP_HOME ; \
+RUN apk --update add --virtual build-dependencies \
+    build-base \
+    curl-dev \
+    openssl-dev \
+    postgresql-dev \
+    zlib-dev \
+    libxml2-dev \
+    libxslt-dev \
+    libffi-dev \
+    libgcrypt-dev && \
+    cd $APP_HOME && \
     bundle config --global build.nokogiri  "--use-system-libraries" && \
     bundle config --global build.nokogumbo "--use-system-libraries" && \
     bundle config --global build.ffi  "--use-system-libraries" && \
+    bundle config --global build.ruby-audio --with-cflags=-Wno-error=format-security && \
```

Comment:
it doesn't, good catch. I left that in thinking it might make sense to standardize dockerfile across apps, but nvm bad idea.